src/components/stepforms/StepFormHeader.vue

```diff
@@ -0,0 +1,77 @@
+<template functional>
+  <div class="step-edit-form">
+    <button class="step-edit-form__back-button" @click="props.cancel()">
+      <i class="fas fa-angle-left" />
+      BACK
+    </button>
+    <div class="step-edit-form__title-container">
+      <h1>{{ props.title }}</h1>
+      <a :href="`https://weaverbird.toucantoco.com/docs/${props.stepName}`" target="_blank">
+        <i class="fas fa-question-circle" />
+      </a>
+    </div>
+    <div class="step-edit-form__empty" />
+  </div>
+</template>
+<script lang="ts">
+export default {};
```

Comment:
Is it really necessary to be importable?

Reply:
Not at all my friend :) Thanks for the review